### PR TITLE
Update copy on ‘terms of use’ page

### PR DESCRIPTION
**Pull request by TobyRet**
## Context

```
As a...candidate
I need to...understand how to use the service and what is accepted/unaccepted behaviour
So that... we have a clear set of terms of use that can be deployed to the service
```

## Changes proposed in this pull request

Update copy on _terms of use_ page

## Link to Trello card

https://trello.com/c/bkkOP14O/2724-dev-terms-of-use

## Things to check

- [x] This code does not rely on migrations in the same Pull Request
- [ ] If this code includes a migration adding or changing columns, it also backfills existing records for consistency
- [ ] API release notes have been updated if necessary
- [ ] New environment variables have been [added to the Azure config](https://github.com/DFE-Digital/apply-for-teacher-training#azure-hosting-devops-pipeline)

### CODE REVIEW

File: app/views/content/terms_candidate.md [hunk 1]
```diff
@@ -1,45 +1,76 @@
...
+The Department of Education does not decide on your application. The course provider does.
+
+There are deadlines that you and the course provider will need to meet. We’ll tell you what they are when you apply. If you, or the course provider, don’t respond in time your application might be deleted.
 
 ## How we check your suitability for working with children
 
 All candidates must have an enhanced DBS check. This will show up any spent or unspent criminal convictions.
 
-Not all convictions will stop you from teaching. [Talk to your provider about their policy on criminal convictions](https://www.gov.uk/exoffenders-and-employment).
+Not all convictions will stop you from teaching. [Talk to your provider about their policy on criminal convictions](https://www.gov.uk/exoffenders-and-employment) .
```
Comment (paulrobertlloyd) [inline app/views/content/terms_candidate.md:56]: Errant space

```suggestion
Not all convictions will stop you from teaching. [Talk to your provider about their policy on criminal convictions](https://www.gov.uk/exoffenders-and-employment).
```

File: app/views/content/terms_candidate.md [hunk 2]
```diff
@@ -1,40 +1,53 @@
-Thank you for using Apply for teacher training, a service run by the Department for Education.
+Terms are a mixture of rules and things to know, like what we do with your data. Applying for teacher training here means you agree to our terms.
 
-Using Apply for teacher training means you agree to these terms of use.
+If you don’t follow them your application might be withdrawn.
```
Comment (gregknight1) [inline app/views/content/terms_candidate.md:3]: We should avoid negative contractions for consistency.

Change 'don't' to 'do not'

File: app/views/content/terms_candidate.md [hunk 3]
```diff
@@ -1,40 +1,53 @@
...
+You can withdraw your application at any point, even after you have accepted an offer.
+
+[Sign in to your account](https://qa.apply-for-teacher-training.service.gov.uk/candidate/sign-in) and click ‘withdraw’ next to the course you want to withdraw.
 
-[Sign in to your account](/candidate/sign-in) and click ‘withdraw’ next to the course(s) you want to withdraw.
+### Outcome of your application
+
+The Department of Education does not decide on your application. The course provider does.
+
+There are deadlines that you and the course provider will need to meet. We’ll tell you what they are when you apply. If you, or the course provider, don’t respond in time your application might be deleted.
```
Comment (gregknight1) [inline app/views/content/terms_candidate.md:50]: We should avoid negative contractions for consistency.

Change 'don't' to 'do not'

File: app/views/content/terms_candidate.md [hunk 4]
```diff
@@ -43,3 +56,21 @@ All candidates must have an enhanced DBS check. This will show up any spent or u
...
+
+## Changing our service
+
+Our service is always evolving, and we will make changes from time to time. We won’t tell you about the changes beforehand. We will let you know if the website will be unavailable while we’re updating it.
+
+## About your online account
+
+When you start your application, we’ll create an account for you automatically. Any information you give will be stored there. We’ll keep your data safe and only use it in line with our [privacy policy](https://www.apply-for-teacher-training.service.gov.uk/candidate/privacy-policy).
+
+You don’t need a password for your account. Instead, each time you log in, we’ll email you a unique and secure link.
```
Comment (gregknight1) [inline app/views/content/terms_candidate.md:76]: We should avoid negative contractions for consistency.

Change 'don't' to 'do not'

File: app/views/content/terms_candidate.md [hunk 5]
```diff
@@ -43,3 +56,21 @@ All candidates must have an enhanced DBS check. This will show up any spent or u
...
+
+If you need our help, email <becomingateacher@digital.education.gov.uk>.
+
+We respond within 5 working days, or 1 working day for more urgent queries.
+
+If you’re applying from overseas, you can find out more about becoming a teacher in England in our guidance for overseas graduates.
+
+## Changing our service
+
+Our service is always evolving, and we will make changes from time to time. We won’t tell you about the changes beforehand. We will let you know if the website will be unavailable while we’re updating it.
```
Comment (gregknight1) [inline app/views/content/terms_candidate.md:70]: We should avoid negative contractions for consistency.

Change 'won't' to 'will not'.

**Review by gregknight1 — CHANGES_REQUESTED** [review]

**Review by gregknight1 — APPROVED** [review]
LGTM

**Pull request merged**